src/unix/qnx.c

```diff
@@ -67,18 +77,45 @@ int uv_exepath(char* buffer, size_t* size) {
 }
 
 
+static uint64_t uv__read_pidin_info(const char* what) {
+  uint64_t rc;
+  char* p;
+  char buf[BUF_SIZE];
+  const char* cmd = "pidin info";
+
+  FILE* fp = popen(cmd, "r");
+  if(fp == NULL)
+    return 0;
+
+  size_t sz = fread(buf, 1, sizeof(buf) - 1, fp);
+  buf[sz] = '\0';
+
+  pclose(fp);
+
+  p = strstr(buf, what);
+  if(p == NULL)
+    return 0;
+
+  p += strlen(what);
+
+  rc = 0;
+  sscanf(p, "%" PRIu64 " MB", &rc);
+
+  return rc * CONVERT_MB;
```

Comment:
I'd just write this as:
```suggestion
  return rc * 1024 * 1024;
```
Clearer, less indirection.

Reply:
Makes sense, changed it to make cleaner!